### ISSUE TITLE
Not working with clear install

### ISSUE THREAD

**Issue by jaswdr** [vluzrmos/lumen-cors#4]
I've installed on a fresh copy of lumen but doesn't works, trace:

405 - MethodNotAllowedHttpException

```
in Application.php line 1176
at Application->handleDispatcherResponse(array('2', array('POST', 'GET'))) in Application.php line 1131
at Application->Laravel\Lumen\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 139
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in CorsMiddleware.php line 26
at CorsMiddleware->handle(object(Request), object(Closure))
at call_user_func_array(array(object(CorsMiddleware), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in StartSession.php line 61
at StartSession->handle(object(Request), object(Closure))
at call_user_func_array(array(object(StartSession), 'handle'), array(object(Request), object(Closure))) in Pipeline.php line 124
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in Pipeline.php line 103
at Pipeline->then(object(Closure)) in Application.php line 1352
at Application->sendThroughPipeline(array('Illuminate\Session\Middleware\StartSession', 'Vluzrmos\LumenCors\Middlewares\CorsMiddleware'), object(Closure)) in Application.php line 1132
at Application->dispatch(null) in Application.php line 1072
at Application->run() in index.php line 28
at require_once('E:\projetos\testeCors\public\index.php') in server.php line 12
```

I test with a simple OPTIONS request to the server and Lumen 5.1

**Comment by vluzrmos** [comment]
All done!

Thanks for that!

Upgrade the package to 1.0.7

**Comment by vluzrmos** [comment]
Right now, that issue and #2 are fixed on 1.0.9 and will be no longer necessary the service provider, then I released v1.1.0, without that.
